neptunecontrib/monitoring/keras.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
+import neptune
+from neptune.exceptions import LibraryNotInstalled, NeptuneException
+
+try:
+    from keras.callbacks import Callback
+except ImportError:
+    try:
+        from tensorflow.keras.callbacks import Callback
+    except ImportError:
+        raise LibraryNotInstalled('Keras')
```

Comment:
Suppose the user has both tensorflow and keras installed. By default, we import keras.Callback. But if the user then uses NeptuneMonitor with tensorflow.keras, they will get the error https://github.com/keras-team/keras/issues/14125

Reply:
Actually, the fix might be as simple as trying `from tensorflow.keras.callbacks import Callback` first. Again, assuming there are no other imcompatibiilties.